specification/hugr.md

```diff
@@ -1259,6 +1261,7 @@ The new hugr is then derived as follows:
     For each child $C$ of $P$ (in order), if $C \in S$, redirect the hierarchy edge $P \rightarrow C$ to
     target the next node in $N$. Stop if there are no more nodes in $N$.
     Add any remaining nodes in $N$ to the end of $P$'s list of children.
+    (Reindexing may be necessary after this step.)
```

Comment:
Not sure what reindexing this means. The new nodes $N$ are already the *copies made in $\Gamma$*, so that is the new indices. And things like "input/output nodes are first/second children" describes the ordering of the hierarchy edges not the *node* indices.

Reindexing might be necessary after step 7 ??

Reply:
Ah I think that may be what I meant...

Reply:
... however I am struggling to see why it would be necessary there. @willsimmons1465 could you clarify this please, it made sense when we discussed it but I seem to have forgotten now?

Reply:
Forgot they had to be densely indexed. OK will do that.